src/buffer/out/textBuffer.cpp

```diff
@@ -2403,6 +2456,34 @@ HRESULT TextBuffer::Reflow(TextBuffer& oldBuffer,
             }
         }
     }
+
+    // Finish copying buffer attributes to remaining rows below the last
+    // printable character. This is to fix the `color 2f` scenario, where you
+    // change the buffer colors then resize and everything below the past

```

Comment:
```suggestion
    // change the buffer colors then resize and everything below the last
```

Reply:
this comment suggests we'll have to inbox this change to fix `COLOR`

Reply:
yea, we would.